app/controllers/steps/evidence/upload_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Steps
+  module Evidence
+    class UploadController < Steps::EvidenceStepController
+      def edit
+        @form_object = UploadForm.build(
+          current_crime_application
+        )
+      end
+
+      # :nocov:
+      def update
```

Comment:
Upload will not happen here, this will advance to next step, uploads have happened before this.
This however will be used to update the categories of the files. But that is for later.